README.md

```diff
@@ -7,17 +7,36 @@ govet examines Go source code and reports suspicious constructs, such as Printf
 ### Installation
 
 1. If you haven't already, [install the Code Climate CLI](https://github.com/codeclimate/codeclimate).
-2. Run `codeclimate engines:enable govet`. This command both installs the engine and enables it in your `.codeclimate.yml` file.
-3. You're ready to analyze! Browse into your project's folder and run `codeclimate analyze`.
+2. Add the following to yout Code Climate config:
+  ```yaml
+  engines:
```

Comment:
Please use the new v2 schema for examples now, which uses the `plugins:` key instead of `engines:`.